index.js

```diff
@@ -10,125 +10,127 @@ import {
   deleteTaskInputCard,
   initialTaskContainer,
   tabs,
+  loadMoreButton,
 } from "./scripts/elements.js";
 import { handleCreateTodo, resetTaskInput } from "./scripts/addTask.js";
 import { todos } from "./scripts/deteleTask.js";
 import { createTaskElement } from "./scripts/taskActions.js";
 import { validateInput } from "./scripts/utilities.js";
-import { COMPLETE, INCOMPLETE } from "./const.js";
-
-export let filteredTasks = [];
-export let filteredTodos = [];
-
-export const handleTabClick = (event) => {
-  const selectedValue = event.target.id;
-  switch (selectedValue) {
-    case COMPLETE:
-      renderTasksByStatus(true);
-      break;
-    case INCOMPLETE:
-      renderTasksByStatus(false);
-      break;
-    default:
-      renderTodoList();
+import { TASK_PER_PAGE, START_INDEX, ALL, COMPLETE } from "./const.js";
+
+let searchText = "";
+let filterStatus;
+let searchResult;
```

Comment:
We don't need this globally. Please make sure to understand which state is global and which state is component state. It'll be very important later down the road specially when working with frameworks based around components.

```suggestion

```